templates/bridge-interface.j2

```diff
@@ -11,9 +11,14 @@ iface {{ openvpn_dev }} inet manual
 # Bridge
 auto br-{{ openvpn_dev }}
 iface br-{{ openvpn_dev }} inet static
-       bridge_ports {{ openvpn_dev }}
+       bridge_ports {% if 'ports' in openvpn_bridge %}{{ openvpn_bridge.ports }}
```

Comment:
*(this is just a style related comment)*
even if it would be long, I would prefer having everything on oneline, the resulting configuration would probably be better, although I would have to test it, which I haven't yet. Have you tested it and if yes does the resulting configuration file look ok and works?